README.md

```diff
@@ -1,17 +1,19 @@
-# Puppeteer Recorder
+# Headless Recorder (previously Puppeteer Recorder)
 
 [![Build Status](https://travis-ci.org/checkly/puppeteer-recorder.svg?branch=develop)](https://travis-ci.org/checkly/puppeteer-recorder)
 
 #### Gold sponsor
-[<img src="chrome-store/checkly-logo.png?raw=true" alt="Checkly" width="300">](https://checklyhq.com?utm_source=github&utm_medium=sponsor-logo-github&utm_campaign=puppeteer-recorder)
+[<img src="chrome-store/checkly-logo.png?raw=true" alt="Checkly" width="300">](https://checklyhq.com?utm_source=github&utm_medium=sponsor-logo-github&utm_campaign=headless-recorder)
 
 ![](src/images/recorder.png)
 
-⭐️ We just published an update on the glorious future of Puppeteer Recorder, [check out this pinned issue](https://github.com/checkly/puppeteer-recorder/issues/83) ⭐️ 
+⭐️ Puppeteer Recorder is now **Headless Recorder** and supports Playwright! ⭐️   
 
-Puppeteer recorder is a Chrome extension that records your browser interactions and generates a
-[Puppeteer](https://github.com/GoogleChrome/puppeteer) script. Install it from the [Chrome Webstore](https://chrome.google.com/webstore/detail/puppeteer-recorder/djeegiggegleadkkbgopoonhjimgehda).
-This project is pretty fresh, but does the following already:
+Headless recorder is a Chrome extension that records your browser interactions and generates a
+[Puppeteer](http://pptr.dev/) or [Playwright](https://playwright.dev/) script. Install it from the [Chrome Webstore](https://chrome.google.com/webstore/detail/puppeteer-recorder/djeegiggegleadkkbgopoonhjimgehda).
```

Comment:
does the chrome store webpage will change? now is `puppeteer-recorder/`?

Reply:
@maxigimenez as far as I could investigate, yes. It will change the name, but keep the unique ID.